include/sparrow/data_traits.hpp

```diff
@@ -18,6 +18,8 @@
 #include "fixed_size_layout.hpp"
 #include "variable_size_binary_layout.hpp"
 
+using time_point = std::chrono::time_point<std::chrono::system_clock>;
```

Comment:
Do you really want to do it outside the sparrow namespace ?

Reply:
Indeed, we probably should have it part of the `sparrow` namespace.